docs/user/security/authentication/index.asciidoc

```diff
@@ -145,43 +194,52 @@ xpack.security.authc.saml.maxRedirectURLSize: 1kb
+  oidc.oidc1:
+    order: 0
+    realm: oidc1
 --------------------------------------------------------------------------------
 
-. {kib} needs to specify which OpenID Connect realm in {es} should be used, in case there are more than one configured there.
-+
-[source,yaml]
---------------------------------------------------------------------------------
-xpack.security.authc.oidc.realm: oidc1
---------------------------------------------------------------------------------
+If you want to use Third Party initiated Single Sign-On, configure your OpenID Provider to use `/api/security/oidc/initiate_login` as `Initiate Login URI`.
+
+It's also possible to configure multiple OpenID Connect authentication providers at the same time. In this case users will need to choose which provider to use for login at the Login Selector screen:
 
-. {kib} supports Third Party initiated Single Sign On, which might start with an external application instructing the user's
-browser to perform a "non-safe" `POST` HTTP method. This request will not include CSRF protection HTTP headers that are
-required by {kib}. If you want to use Third Party initiated SSO , then you must disable the CSRF check for this endpoint.
-+
 [source,yaml]
 --------------------------------------------------------------------------------
-server.xsrf.whitelist: [/api/security/oidc/initiate_login]
+xpack.security.authc.providers:
+  oidc.oidc1:
+    order: 0
+    realm: oidc1
+    description: "Log in with Elastic"
+  oidc.oidc2:
+    order: 1
+    realm: oidc2
+    description: "Log in with Auth0"
 --------------------------------------------------------------------------------
 
 [float]
 ===== OpenID Connect and basic authentication
 
-Similar to SAML, OpenID Connect support in {kib} is designed to be the primary (or sole) authentication method for users
-of that {kib} instance. However, you can configure both OpenID Connect and Basic authentication for the same {kib} instance:
+Similar to SAML, OpenID Connect support in {kib} is designed to be the primary (or sole) authentication method for users of that {kib} instance. However, you can configure both OpenID Connect and Basic authentication for the same {kib} instance. This might be the case for {kib} or {es} admins whose accounts aren't linked to the Single Sign-On users database:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-xpack.security.authc.providers: [oidc, basic]
+xpack.security.authc.providers:
+  oidc.oidc1:
+    order: 0
+    realm: oidc1
+    description: "Log in with Elastic"
+  basic.basic1:
+    order: 1
 --------------------------------------------------------------------------------
 
-Users will be able to access the login page and use Basic authentication by navigating to the `/login` URL.
+Basic authentication is supported _only_ if `basic` authentication provider is explicitly declared in `xpack.security.authc.providers` setting in addition to `oidc`.
```

Comment:
`explicitly declared in `xpack.security.authc.providers` setting in addition to `oidc`.` > `explicitly declared in the `xpack.security.authc.providers` setting, in addition to `oidc`.`